cmd/proxy/main.go

```diff
@@ -138,6 +170,31 @@ func main() {
 }
 
 func startProxy(ctx *cli.Context) error {
+	logLevelFlagValue := ctx.GlobalString(logLevel.Name)
```

Comment:
you can extract the logger handling code in a new function

Reply:
done.